qa/integration-tests/pom.xml

```diff
@@ -45,12 +62,22 @@
       <artifactId>camunda-qa-util</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Not sure if still needed since ExecutionQueue is not an Actor anymore 🤔

Reply:
The mvn check told me that we need it. :D